roles/kubernetes-apps/ansible/tasks/cleanup_dns.yml

```diff
@@ -54,3 +54,13 @@
     - kubeadm_init is defined
     - kubeadm_init.changed|default(false)
     - inventory_hostname == groups['kube-master'][0]
+
```

Comment:
Maybe removing the remote /etc/kubernetes/kubedns-autoscaler.yml would be fine

Reply:
Well, it is not auto added. The .yml is just there. The cleanup for KubeDNS and CoreDNS does not remove .yml files on the master either. It only deleted the kube deployments, services etc.